ui/pages/confirm-transaction-base/gas-details-item/gas-details-item.js

```diff
@@ -29,6 +30,9 @@ const GasDetailItem = ({
   useNativeCurrencyAsPrimaryCurrency,
 }) => {
   const t = useI18nContext();
+  const { estimateToUse } = useGasFeeContext();
+  console.log('estimateToUse = ', estimateToUse); // estimateToUse available for use here
```

Comment:
This console.log can be removed

Reply:
it is removed in following PR actually, I had added it here just for demoing the use of context in POC

let me remove here also